nginx.conf.sample

```diff
@@ -198,6 +203,6 @@ gzip_types
 gzip_vary on;
 
 # Banned locations (only reached if the earlier PHP entry point regexes don't match)
-location ~* (\.php$|\.htaccess$|\.git) {
+location ~* (\.php$|\.phtml$|\.htaccess$|\.git) {
```

Comment:
@schmengler maybe forbid both `xml` and `phtml` in both occurrences here?

Reply:
I was tempted to, but not sure if that could block legit requests

Reply:
wouldn't that block sitemaps?

Reply:
@schmengler, ok, noticed

> For Nginx: deny access to PHTML files in general and XML files within errors directory (nginx.conf.sample)

Do we need to give access for some `xml` in reality?

Reply:
Thanks @wigman!) That's the case.

Reply:
@wigman @schmengler but maybe we can disable all XML but enable sitemaps explicitly?

Reply:
although they should not be, sitemaps can be named dynamically so this is not possible I guess?
so for now maybe only excluding .xml in the errors directory could do it?

Reply:
@davidverholen aren't they placed in some specific directory with name corresponding to some mask?

Reply:
you can choose a relative path and filename under the pub directory, so it's not possible to determine (at least by filename)

Reply:
maybe a solution could be, to block the errors directory completely and only explicitly allow files that need to be public? Could be difficult for custom error themes as they lie in sub directories of errors and also can be named dynamically